pkg/controller/v1alpha2/experiment/experiment_controller.go

```diff
@@ -184,8 +184,10 @@ func (r *ReconcileExperiment) Reconcile(request reconcile.Request) (reconcile.Re
 
 	}
 	if !instance.IsCreated() {
+		//Get Experiment config from instance
+		experimentConfig := util.GetExperimentConf(instance)
 		//Experiment not created in DB
-		err = util.CreateExperimentInDB(instance)
+		err = util.CreateExperimentInDB(experimentConfig)
```

Comment:
As we would like to remove dependency of api manager from controller, I would recommend to keep grpc api related config out of the controller code.  Let util take care of getting the config and calling the API

Reply:
I put it inside `CreateExperimentInDB` function in `util`. Is it fine?